crates/runtime/src/dataconnector/github/mod.rs

```diff
@@ -510,14 +529,35 @@ lazy_static! {
                     Operator::Gt,
                     Operator::GtEq,
                 ],
-                remap: Some("merged"),
+                remaps: Some(vec![GitHubFilterRemap::Column("merged")]),
+                uses_modifiers: true
+            },
+        );
+
+        m.insert(
+            "committed_date",
```

Comment:
Can you create an issue to update the Github docs to cover this? The `Spec` section from your Issue covers a good chunk of what we'd need

Reply:
https://github.com/spiceai/docs/issues/435